tests/InArrayValidatorTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace ipl\Tests\Validator;
+
+use ipl\I18n\NoopTranslator;
+use ipl\I18n\StaticTranslator;
+use ipl\Validator\InArrayValidator;
+
+class InArrayValidatorTest extends TestCase
+{
+    public function testWithValidValue()
```

Comment:
You test what with valid value 😆? I would just reduce the method names to `testValidValues`, `testInvalidValues` and so on.